### PR TITLE
Booking counter

### PR DESCRIPTION
**Pull request by oliveiraigorm**
### Changes

Writes to the app settings the amount of internal and external bookings

### Testing

Book new sessions from the dashboard and by using the external embed link in another browser or when you aren't logged in.
To see the updated counter more easily this can be tested in combination with https://github.com/MetaProvide/adminly_core/pull/116

### CODE REVIEW

File: lib/Controller/PageController.php [hunk 1]
```diff
@@ -151,6 +151,22 @@ public function formPostEmb() {
 
         $tr = $this->showFormPost($userId, $pageId, true);
         $this->setEmbCsp($tr, $userId);
+
+        $hasActiveValidSession = false;
+        
+        if($this->userSession->getUser()){
+            $hasActiveValidSession = $this->userSession->getUser()->getUid() == $userId;
```
Comment (mwalbeck) [inline lib/Controller/PageController.php]: Wouldn't this count a user who is using the external widget but is logged in as an internalWidgetBooking?

Comment (oliveiraigorm) [reply lib/Controller/PageController.php]: Yes, good catch. I've changed the logic to use js to better detect it

Comment (mwalbeck) [reply lib/Controller/PageController.php]: I think we need to double-check what the intention is behind the metric and what is considered internal. With the new implementation, if you access public booking form link and book a session it's logged as an internal booking, but I'm unsure whether that should count or it's only booking made from the dashboard that are considered internal.

Comment (mwalbeck) [reply lib/Controller/PageController.php]: Just checking the original description it does say bookings made on the dashboard as internal, so the current implementation needs to change.

Comment (oliveiraigorm) [reply lib/Controller/PageController.php]: > I think we need to double-check what the intention is behind the metric and what is considered internal. With the new implementation, if you access public booking form link and book a session it's logged as an internal booking, but I'm unsure whether that should count or it's only booking made from the dashboard that are considered internal.

The logging only happens in the controller function that handles the post request from the embed forms so the public page wouldn't be logged.

Comment (mwalbeck) [reply lib/Controller/PageController.php]: Great!

**Review by mwalbeck — APPROVED** [review]

**Pull request merged**